infrastructure/server/aurora-v2.tf

```diff
@@ -167,6 +168,40 @@ resource "aws_secretsmanager_secret_version" "famdb_apicreds_secret_version" {
 EOF
 }
 
+# Create Auth Lambda DB User Credentials
+
+variable "famdb_auth_lambda_user" {
```

Comment:
The lamdba function resource name was suffixed with the environment. Should we do that for all resources by default?

Reply:
It was a recommendation on the "rename" ticket. Doing this for all resources seems like a lot of change for no benefit. Definitely don't want to recreate the DB with a different name (data restore in prod is hard).